ow_regolith/README.md

```diff
@@ -0,0 +1,129 @@
+regolith model is spawned in the scoop and the tracked total volume has the 
+threshold deducted from it.
+
+Spawning is done by calling the `/gazebo/spawn_sdf_model` ROS service, followed
+by a call to `/gazebo/apply_body_wrench`, so that the model is kept in the scoop
+with a *fake force* of a magnitude that's just enough to keep the particle from 
+rolling out. Upon completion of a dig action, the *fake force* is removed from
+all regolith models, so they may settle within the scoop and behave like normal
+regolith material during any following arm movements.
+
+All regolith models spawned by this node are removed from the Gazebo world upon
+completion of the delivery arm action. Future versions of this package will 
+incorporate smarter logic around when to clean-up regolith models.
+
+## Caveats
+
+- The majority of models spawned by `regolith_node` will wind up firmly held in
+the scoop, but at time of writing 1 out of every 4 may fall out and be lost in
+the collision geometry below. They will, however, still be cleaned up by the
+node when a delivery arm action is performed.
+- The fake force applied to particles during any dig operation will be 
+transferred to the joints of the arm, but the magnitude of the force itself is 
+not necessarily realistic to what a digging scoop may experience. Keep this in 
+mind when using this plug-in for any study that records forces on the arm.
+- This node will not work properly if the user calls the ROS Service versions of
+arm operations, instead of their action counterparts. The ROS Service arm 
+operations are deprecated in this simulator, and should be avoided anyways.
+- While sample is in the scoop you may see the following error flood the console
+```ODE Message 3: LCP internal error, s <= 0 (s=0.0000e+00)```
+We're still looking into how to avoid or suppress this message. For now it just
+has to be ignored.
+
+## Usage
+
+### Launch File
+
+The `regolith_node` may be added to a launch file like so
+```xml
+<node name="regolith_node" pkg="ow_regolith" type="regolith_node" output="screen">
+    <param name="spawn_volume_threshold" type="double" value="1e-3"/>
+    <param name="regolith_model_uri"     type="string" value="model://ball_icefrag_2cm"/>
+</node>
+```
+The two parameters, `spawn_volume_threshold` and `regolith_model_uri`, are
+required by the node, and there will be an error printed if it is ran without
+these two being set. 
+
+`spawn_volume_threshold` is the cubic meters that must be removed from the
+terrain before a regolith model is spawned. It may have to be fine-tuned if a 
+regolith model of larger size is used in place of the default to avoid 
+consecutively spawned models colliding with each other.
+
+`regolith_model_uri` tells the node which model out of the Gazebo model database
+should be spawned each time the `spawn_volume_threshold` is reached.
+
+### ROS Service
+
+ROS Service support for this node is still to come...
```

Comment:
Suggest simply saying that ROS Services are not supported at this time.  I'm not sure if and when we will prioritize further ROS Service work.